src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -111,6 +116,12 @@
   private byte[] uid;
   private SecureChannel secureChannel;
 
+  static final short CERT_LEN = 71;
+  static final short NUM_CERTS = 3;
+  static final short CERTS_LEN = CERT_LEN * NUM_CERTS;
```

Comment:
currently value is 71 * 3 = <u>213</u>, must never exceed 255.

Reply:
Yep, it won't